CONTRIBUTING.md

```diff
@@ -32,6 +32,25 @@ A good bug report will help direct developers to solve the problem at hand witho
 
 If you have a budding idea or a feature that requires a more community-involved discussion, consider having the development discussion on the [live chat](https://riot.im/app/#/room/#CiviWiki:matrix.org) or create a thread on [loomio](https://www.loomio.org/g/ET40tXUC/openciviwiki). This will allow for a well-thought-out issue that will more likely be in line with the goal of the project.
 
+### Solve Conflicts with code 
```

Comment:
This would be better as a sub-section of `## Development`, since it is intended for software developers. Add a `## Troubleshooting` section after `## Development` and place this under `## Troubleshooting` as `### Solve conflicts with poetry.yaml`

Reply:
Note, the `## Development` section is quite long, so look for the next level-two heading (`##`).